.gitignore

```diff
@@ -18,3 +18,4 @@
 
 # Ignore application configuration
 /config/application.yml
+public/uploads
```

Comment:
This is a nice to have that will ignore the `public/uploads` file, so you don't check in the photos you upload on localhost to GitHub.  It will keep your GitHub a little cleaner.